### DISCUSSION TITLE
only part of the image is processed

### DISCUSSION THREAD

**Discussion by alwadey** [davidfrantz/force#200] [Q&A] [answered]
I have processed a sentinel-2 image named T38RPN
![prosessed](https://user-images.githubusercontent.com/35626030/179200240-3c49c835-c331-4f94-a7d1-e62461bad0f5.png)
 to level-2 but the output was only part of this image and not all the image.
the parameter file is :
++PARAM_LEVEL2_START++

# INPUT/OUTPUT DIRECTORIES
# ------------------------------------------------------------------------
FILE_QUEUE = /home/alwadey/force/level1/sentinel/queue.txt
DIR_LEVEL2 = /home/alwadey/force/level2
# This is the directory where logfiles should be saved.
# Type: full directory path
DIR_LOG = /home/alwadey/force/log
DIR_TEMP = /home/alwadey/force/temp

# DIGITAL ELEVATION MODEL
# ------------------------------------------------------------------------
FILE_DEM = /home/alwadey/force/mis/DEM/n24_e046_3arc_v2.tif
# Nodata value of the DEM.
# Type: Integer. Valid range: [-32768,32767]
DEM_NODATA = -32767

# DATA CUBES
# ------------------------------------------------------------------------
DO_REPROJ = FALSE
DO_TILE = FALSE
FILE_TILE = NULL
TILE_SIZE = 30000
BLOCK_SIZE = 3000
RESOLUTION_LANDSAT = 30
RESOLUTION_SENTINEL2 = 10
ORIGIN_LON = -25
ORIGIN_LAT = 60
PROJECTION = NULL
# This is the resampling option for the reprojection; you can choose
# between Nearest Neighbor (NN), Bilinear (BL) and Cubic Convolution
# (CC); not used if DO_REPROJ = FALSE.
# Type: Character. Valid values: {NN,BL,CC}
RESAMPLING = CC

# RADIOMETRIC CORRECTION OPTIONS
# ------------------------------------------------------------------------
DO_ATMO = TRUE
DO_TOPO = TRUE
DO_BRDF = TRUE
ADJACENCY_EFFECT = TRUE
MULTI_SCATTERING = TRUE

# WATER VAPOR CORRECTION OPTIONS
# ------------------------------------------------------------------------
DIR_WVPLUT = NULL
WATER_VAPOR = NULL

# AEROSOL OPTICAL DEPTH OPTIONS
# ------------------------------------------------------------------------
DO_AOD  = TRUE
DIR_AOD  = NULL

# CLOUD DETECTION OPTIONS
# ------------------------------------------------------------------------
ERASE_CLOUDS = FALSE
MAX_CLOUD_COVER_FRAME = 75
MAX_CLOUD_COVER_TILE  = 75
CLOUD_BUFFER  = 300
SHADOW_BUFFER = 90
SNOW_BUFFER   = 30
CLOUD_THRESHOLD  = 0.225
SHADOW_THRESHOLD = 0.02

# RESOLUTION MERGING
# ------------------------------------------------------------------------
RES_MERGE = IMPROPHE

# CO-REGISTRATION OPTIONS
# ------------------------------------------------------------------------

DIR_COREG_BASE = NULL

COREG_BASE_NODATA = -9999

# MISCELLANEOUS OPTIONS
# ------------------------------------------------------------------------

IMPULSE_NOISE = FALSE
# This parameter defines if nodata pixels should be buffered by 1 pixel.
# Type: Logical. Valid values: {TRUE,FALSE}
BUFFER_NODATA = FALSE

# TIER LEVEL
# ------------------------------------------------------------------------

TIER = 1

# PARALLEL PROCESSING
# ------------------------------------------------------------------------
NPROC = 1
NTHREAD = 8
PARALLEL_READS = FALSE
DELAY = 30
TIMEOUT_ZIP = 60

# OUTPUT OPTIONS
# ------------------------------------------------------------------------

OUTPUT_FORMAT = GTiff
OUTPUT_DST = FALSE
OUTPUT_AOD = FALSE
OUTPUT_WVP = FALSE
OUTPUT_VZN = FALSE
OUTPUT_HOT = FALSE
OUTPUT_OVV = TRUE

++PARAM_LEVEL2_END++

**Comment by ernstste** [comment, marked as answer]
T38RPN is the tile name, what is the complete image name?
Depending on which orbit the image was acquired in, it's perfectly normal if it only covers part of the MGRS tile.

**Reply by alwadey** [reply]
I mean processed the tile T38RPN but only part of it was process which I attached. why all the tile was not processed?
S2B_MSIL1C_20220621T071619_N0400_R006_T38RPN_20220621T080106

**Reply by ernstste** [reply]
If you open the image before processing, you will see that it has the same coverage. This is not related to FORCE but simply how the tiling for Sentinel-2 works.
Sentinel-2 orbits cover MGRS tiles either in full or in part. The tile that you chose for analysis (T38RPN) is only partly covered by orbit track 6 which the image you're processing was acquired in. If you need full coverage, you want to use an image from the neighboring track to the West (49). 

You can download the Sentinel-2 MGRS grid and orbits here. Overlay them and have a look for yourself if you like:
https://github.com/ernstste/ls_s2_tiles/blob/main/ls_s2_tiles.zip
https://sentinel.esa.int/documents/247904/685211/S2A_rel_orbit_groundtrack_10Sec/57bcb79f-2696-4859-8292-07ac7166e884

**Reply by alwadey** [reply]
thanks, that is clear now

**Reply by alwadey** [reply]
so is there any way to control the download by orbit number?

**Reply by ernstste** [reply]
This is not implemented in FORCE-level1-csd, but you can filter by orbit track if you use the Copernicus Open Access Hub interface (https://scihub.copernicus.eu/dhus/#/home).
